be/src/types/constexpr.h

```diff
@@ -28,6 +28,9 @@ constexpr int HLL_COLUMN_DEFAULT_LEN = HLL_REGISTERS_COUNT + 1;
 // 1 for type; 1 for hash values count; 8 for hash value
 constexpr int HLL_EMPTY_SIZE = 1;
 
+const static int MAX_HLL_LOG_K = 20;
+const static int HLL_LOG_K = 17;
```

Comment:
Well done~ can`HLL_LOG_K ` be configured as a param? like presto's implements: https://prestodb.io/docs/current/functions/hyperloglog.html

Reply:
Tried to find out how to implement parameterized precision, but in the current function framework, aggregate function is recognized by the following parameters:
```
    const AggregateFunction* get_aggregate_info(const std::string& name, const LogicalType arg_type,
                                                const LogicalType return_type, const bool is_window_function,
                                                const bool is_null)
```
I can't pick the right function with HLL precision at the same time.

And I either can't initialize HLL structure at function runtime, because HLL structure works as `AggData`, it only has a default constructor without any parameters.

Any suggestions, thank you.